supreme/src/androidMain/kotlin/at/asitplus/signum/supreme/os/AndroidKeyStoreProvider.kt

```diff
@@ -295,7 +295,7 @@ sealed class AndroidKeystoreSigner private constructor(
 ) : PlatformSigningProviderSigner<AndroidSignerSigningConfiguration, AndroidKeystoreAttestation> {
 
     @SecretExposure
-    override fun exportPrivateKey(): KmmResult<CryptoPrivateKey.WithPublicKey<*>> = KmmResult.failure(IllegalStateException("Non-Exportable key"))
+    override fun exportPrivateKey(): KmmResult<Nothing> = KmmResult.failure(IllegalStateException("Non-Exportable key"))
```

Comment:
i note that i am unhappy with this dynamically failing instead of marking signers with an exportable privatekey somehow... but union types and no good traits and yada yada